griptape/drivers/vector/base_vector_store_driver.py

```diff
@@ -139,6 +139,17 @@ def load_entry(self, vector_id: str, *, namespace: Optional[str] = None) -> Opti
     @abstractmethod
     def load_entries(self, *, namespace: Optional[str] = None) -> list[Entry]: ...
 
+    @abstractmethod
+    def query_vector(
+        self,
+        vector: list[float],
+        *,
+        count: Optional[int] = None,
+        namespace: Optional[str] = None,
+        include_vectors: bool = False,
+        **kwargs,
+    ) -> list[Entry]: ...
+
```

Comment:
Adding abstract method is technically a breaking change for any users who have created their own vector store driver.

What do you think about:
```suggestion
    def query_vector(
        self,
        vector: list[float],
        *,
        count: Optional[int] = None,
        namespace: Optional[str] = None,
        include_vectors: bool = False,
        **kwargs,
    ) -> list[Entry]:
      # TODO: Mark as abstract method for griptape 2.0
      raise NotImplementedError(f"{self.__class__.__name__} does not support vector query.")
````